lib/presentation/register/view/register_page_view.dart

```
@@ -133,90 +138,122 @@ class _RegisterPageView extends State<RegisterPageView> {
           height: 10,
         ),
         CustomTextForm(
-            validator: (value) {
-              if (value == null || value.isEmpty) {
-                return AppLocalizations.of(context)!.empty_name;
-              }
-              name = value;
-              return null;
-            },
-            labelText: AppLocalizations.of(context)!.name,
-            keyboardType: TextInputType.text,
-            isNonPasswordField: true,
-            maxLength: nameMaxLength),
+          validator: (value) {
+            if (value == null || value.isEmpty) {
+              return AppLocalizations.of(context)!.empty_name;
+            }
+            name = value;
+            return null;
+          },
+          labelText: AppLocalizations.of(context)!.name,
+          keyboardType: TextInputType.text,
+          isPasswordField: false,
+          maxLength: nameMaxLength,
+          obscureText: false,
```

In your custom widget you could assign default value to obscureText to be false